examples/expectations/column_map_expectation_template.py

```diff
@@ -59,6 +59,35 @@ class ExpectColumnValuesToMatchSomeCriteria(ColumnMapExpectation):
     # This dictionary contains default values for any parameters that should have default values
     default_kwarg_values = {}
 
+    def validate_configuration(self, configuration: Optional[ExpectationConfiguration]):
```

Comment:
Gonna need `from great_expectations.exceptions import InvalidExpectationConfigurationError` at the top of the file